lighthouse-core/audits/seo/meta-description.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * @license Copyright 2017 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+const Audit = require('../audit');
+
+class Description extends Audit {
+  /**
+   * @return {!AuditMeta}
+   */
+  static get meta() {
+    return {
+      category: 'Content Best Practices',
```

Comment:
Meta question for the LH team. `category` here is referring to the title of the audit _group_, not to be confused with the top-level audit _category_, SEO, defined in the config file?

https://github.com/GoogleChrome/lighthouse/blob/master/lighthouse-core/config/seo.js#L25-L27

Reply:
@rviscomi yeah the field here has an ambiguous name and is not used anywhere anymore 😆 

I'm in favor of removing entirely now that we have real categories that control quite a bit of report building.